### PR TITLE
docs: add diagram to reference

### PR DESCRIPTION
**Pull request by gonmmarques**
Relates to https://github.com/kubeflow/katib/issues/2442

I was a bit unsure on how the doc should look like and if this is correct, please let me know if I missed something.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/ok-to-test

**Comment by gonmmarques** [comment]
> Thank you for doing this @gonmmarques! Please fix the CI checks.

@andreyvelich  Thank you for the review and the heads-up :) Let me know if I missed something else. 

In the spirit of Hacktoberfest, and if this PR is accepted, please consider adding the label "hacktoberfest-accepted”. Thanks.

File: content/en/docs/components/katib/reference/experiment-cr.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+hyperparameters tuning for user's Machine learning model. The Experiment
+workflow looks as follows:
+
+<center>
+<img width="100%" src="/docs/components/katib/images/katib-workflow.png" alt="Katib Workflow"
+</center>
+
+1. The `Experiment` CR is submitted to the Kubernetes API server. Katib
+   `Experiment` mutating and validating webhook is called to set the default
+   values for the `Experiment` CR and validate the CR separately.
```
Comment (andreyvelich) [inline content/en/docs/components/katib/reference/experiment-cr.md:20]: Please can you make the appropriate changes in all places in this docs to rename `Experiment` CR to Experiment ?
Similar change for `Suggestion` CR and `Trial` CR.

File: content/en/docs/components/katib/reference/experiment-cr.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+## Katib Experiment Lifecycle
+
+When user creates an Experiment, Katib Experiment controller,
+`Suggestion` controller and `Trial` controller is working together to achieve
+hyperparameters tuning for user's Machine learning model. The Experiment
+workflow looks as follows:
+
+<center>
+<img width="100%" src="/docs/components/katib/images/katib-workflow.png" alt="Katib Workflow"
+</center>
```
Comment (andreyvelich) [inline content/en/docs/components/katib/reference/experiment-cr.md:16]: You don't need to put this image in the centre, as you can see all text has been centralized: https://deploy-preview-3906--competent-brattain-de2d6d.netlify.app/docs/components/katib/reference/experiment-cr/#:~:text=The%20Experiment%20is%20submitted%20to%20the%20Kubernetes

Just use the same tags as here for image: https://github.com/kubeflow/website/blame/0d213e8b11efec11b128fe7e4c4832156b252c47/content/en/docs/components/katib/reference/architecture.md#L33-L35

File: content/en/docs/components/katib/reference/experiment-cr.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+1. The `Experiment` is submitted to the Kubernetes API server. Katib
+   `Experiment` mutating and validating webhook is called to set the default
+   values for the `Experiment` and validate the CR separately.
```
Comment (andreyvelich) [inline content/en/docs/components/katib/reference/experiment-cr.md:20]: Please change `Experiment` to Experiment to be consistent.
```suggestion
1. The Experiment is submitted to the Kubernetes API server. Katib
   `Experiment` mutating and validating webhook is called to set the default
   values for the `Experiment` and validate the CR separately.
```

Comment (gonmmarques) [reply content/en/docs/components/katib/reference/experiment-cr.md]: Sorry I had missed this remark.

File: content/en/docs/components/katib/reference/experiment-cr.md (hunk 2 above)
Comment (gonmmarques) [reply content/en/docs/components/katib/reference/experiment-cr.md]: Thanks.

**Comment by andreyvelich** [comment]
@gonmmarques We don't have such label in the website, and I don't think we participate in hacktoberfest this year, isn't ?

**Comment by gonmmarques** [comment]
Hacktoberfest is indeed opt-in for maintainers. To participate, either the repository needs to have the `hacktoberfest` topic or the PR needs to have the `hacktoberfest-accepted` .

Since I saw you added the label `hacktoberfest` on the PR I thought you were wanting to accept into the hacktoberfest.

Thanks anyways.

**Pull request merged**